.github/workflows/release.yml

```diff
@@ -0,0 +1,119 @@
+        description: 'Release version'
+        required: true
+        default: '0.0'
+      devel_name:
+        description: 'Developer Name'
+        required: true
+        default: 'Avocado Developer'
+      devel_mail:
+        description: 'Developer mail'
+        required: true
+        default: 'avocado@redhat.com'
+      rtd_project:
+        description: 'readthedocs project name'
+        required: true
+        default: 'avocado-framework'
+
+jobs:
+
+  release:
+    name: Release pipeline
+    runs-on: ubuntu-latest
+    container:
+      image: fedora:34
+
+    steps:
+      - name: install required packages
+        run:  dnf -y install rpmdevtools git python3-pip make
+      - uses: actions/checkout@v2
+        with:
+          persist-credentials: false
+          fetch-depth: 0
+      - name: Update VERSION files and python-avocado.spec
+        run: |
+          export VERSION=${{ github.event.inputs.version }}
+          export DEVEL_NAME=${{ github.event.inputs.devel_name }}
+          export DEVEL_MAIL=${{ github.event.inputs.devel_mail }}
+          make -f Makefile.gh propagate-version
+          make -f Makefile.gh release-update-spec
+      - name: Commit files and tag
+        run: |
+          export VERSION=${{ github.event.inputs.version }}
+          export DEVEL_NAME=${{ github.event.inputs.devel_name }}
+          export DEVEL_MAIL=${{ github.event.inputs.devel_mail }}
+          git config --local user.email "${{ github.event.inputs.devel_mail }}"
+          git config --local user.name "${{ github.event.inputs.devel_name }}"
+          make -f Makefile.gh release-commit-tag
+      - name: Push changes to github
+        uses: ad-m/github-push-action@master
+        with:
+          github_token: ${{ secrets.RELEASE_TOKEN }}
+          branch: ${{ github.ref }}
+      - name: Build documentation readthedocs
+        env:
+          token: ${{ secrets.RTD_TOKEN }}
+          url: "https://readthedocs.org/api/v3/projects/${{ github.event.inputs.rtd_project }}"
+          version: ${{ github.event.inputs.version }}
+        run: |
+          export VERSION=$version
+          export URL=$url
+          export TOKEN=$token
+          make -f Makefile.gh build-update-readthedocs
+      - run: echo "In a few minutes the release documentation will be available in https://${{ github.event.inputs.rtd_project }}.readthedocs.io/en/${{ github.event.inputs.version }}/"
+      - name: Build wheel
+        run: |
+          make -f Makefile.gh build-wheel
+      - name: Save wheel as artifact
+        uses: actions/upload-artifact@v2
+        with:
+          name: wheel
+          path: /__w/avocado-test/avocado-test/PYPI_UPLOAD/
+          retention-days: 3
+      - name: Upload to pypi
+        env:
+          TWINE_USERNAME: '${{ secrets.PYPI_USER }}'
+          TWINE_PASSWORD: '${{ secrets.PYPI_PASSWD }}'
+        run: |
+          export TWINE_USERNAME=$TWINE_USERNAME
+          export TWINE_PASSWORD=$TWINE_PASSWORD
```

Comment:
Same here: with my previous examples it was not needed the export. Probably I'm missing something.